src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,26 @@
+package duke.task;
+
+import duke.EmptyDescriptionException;
+import duke.NoSpecficTimeException;
+import duke.NoSpecificDeadlineException;
+import duke.task.Task;
+
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class Deadline extends Task {
+    private static String filePath = "test.txt";
+
+    public Deadline(String task) throws EmptyDescriptionException, NoSpecificDeadlineException, NoSpecficTimeException {
+     // Call constructor of super class and change formatting to braces
```

Comment:
As per the Java coding standard, comments should be indented relative to their position in the code.
![image](https://user-images.githubusercontent.com/77711860/190866984-b89d9573-fcb3-4d52-afc9-cb86c388d69f.png)